presets/react/storybook/middleware.js

```diff
@@ -0,0 +1,11 @@
+const { Router } = require('express');
+
+module.exports = ({ pkgPath }) => {
+  const router = Router();
+
+  if (process.env.SSR === 'true') {
```

Comment:
since the whole point of this middleware is to provide SSR the check seems unnecessary

Reply:
`.storybook/middleware.js` is a configuration file from storybook.

- storybook loads packages/components/.storybook/middleware.js 
- always loads `presets/react/storybook/middleware.js`
- then decides if it should use SSR middleware or not

Wouldn't want to move the `env.SSR` check into package's `middleware.js`

Reply:
we always need it though? when is the case we don't want SSR

Reply:
Wouldn't want SSR during normal development. It is way way slower, it has to do full refreshs all the time and the error handling is way nicer on the client. Would rather develop without SSR and check afterwards if it works with SSR enabled